### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by yfzm**
1. force to leave table if short of chips
2. handle table chips correctly

### CODE REVIEW

File: libs/storage.py [hunk 1]
```diff
@@ -157,7 +157,20 @@ def leave_table(self, userid: str, tableid: str, remainChip: int):
             cursor.close()
 
     def change_table_chip(self, userid: str, tableid: str, chips: int):
```
Comment (Veiasai) [inline libs/storage.py]: Initially this function is for avoiding wrong bill when game crashes. we change the table chip every time when executing result. Before the process serving again, it clear all the records of table chip. https://github.com/yfzm/poker-bot/blob/18c712fd485eda6365f6f61e093cc1b2935de720/libs/storage.py#L37

File: libs/game.py [hunk 2]
```diff
@@ -58,9 +62,13 @@ def win_bet(self, player: Player, chip: int):
     def lose_bet(self, player: Player, chip: int):
         self.chip_changes[player] -= chip
 
-    def execute(self):
+    def execute(self, set_storage: Callable[[str, int], Any] = default_set_storage):
```
Comment (Veiasai) [inline libs/game.py]: passing function is terrible.
Why not check all players' chip after execute.

File: libs/storage.py (hunk 1 above)
Comment (yfzm) [reply libs/storage.py]: 所以呢？change_table_chip这个函数一开始是pass，实现的应该没有问题吧

File: libs/game.py [hunk 3]
```diff
@@ -58,9 +62,13 @@ def win_bet(self, player: Player, chip: int):
...
         for player, chip in self.chip_changes.items():
             player.chip += chip
+            if player.chip <= 0:
+                logging.debug("%s has no chips(%d) and is about to leaving", player.username, player.chip)
+                set_storage(player.userid, player.chip)
```
Comment (Veiasai) [inline libs/game.py]: here, should always call set_storage.

File: libs/game.py (hunk 2 above)
Comment (yfzm) [reply libs/game.py]: 有道理，，而且传函数还会让game多处理了不属于它的逻辑，leave和enter都应该在上层处理掉

**Review by Veiasai — APPROVED** [review]

**Pull request merged**